.github/workflows/linting-backend.yml

```
@@ -0,0 +1,27 @@
+# https://deno.land/manual@v1.19.2/continuous_integration
+
+name: CI - Linting - Backend
+
+on:
+  pull_request:
+    paths:
+      - 'backend/**'
+  workflow_dispatch:
+
```

Sets the working directory at file level for all jobs 🚀 
```suggestion

defaults:
  run:
    shell: bash
    working-directory: backend


```